opennms-model/src/main/java/org/opennms/netmgt/model/RrdGraphAttribute.java

```diff
@@ -136,4 +137,19 @@ public String toString() {
     	return ""+m_resource + '.' + m_name;
 	}
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        RrdGraphAttribute that = (RrdGraphAttribute) o;
+        return Objects.equals(m_name, that.m_name) &&
+                Objects.equals(m_relativePath, that.m_relativePath) &&
+                Objects.equals(m_rrdFile, that.m_rrdFile) &&
+                Objects.equals(m_resource, that.m_resource);
```

Comment:
If you call the other side `that` you should prefix the instance with `this.` :-)

```suggestion
        return Objects.equals(this.m_name, that.m_name) &&
                Objects.equals(this.m_relativePath, that.m_relativePath) &&
                Objects.equals(this.m_rrdFile, that.m_rrdFile) &&
                Objects.equals(this.m_resource, that.m_resource);
```

Reply:
:+1: